contracts/SlashIndicator.sol

```diff
@@ -158,6 +188,73 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function handleFinalityReport(FinalityEvidence calldata _evidence) external onlyInit oncePerBlock noReentrant {
```

Comment:
There is no strong requirement about `oncePerBlock`, you can delete it.

Reply:
When we define a function of a smart contract, we stand at the user's view, what about `submitFinalityViolationEvidence`?

Reply:
Fixed